src/__tests__/lib/containers/EntityBadgeIcons.test.tsx

```diff
@@ -312,13 +320,26 @@ describe('EntityBadgeIcons tests', () => {
 
       userEvent.click(unlinkButton)
 
+      // Confirmation modal appears
+      await screen.findByText('Confirm Unlink')
+      userEvent.click(await screen.findByRole('button', { name: 'Unlink' }))
+
       await waitFor(() => expect(defaultProps.onUnlinkError).toBeCalled())
     })
   })
 
   describe('displays schema validity', () => {
```

Comment:
For the tests below
* Ensure MSW reports that a schema is bound, if checking schema validation results
* Adjust what conditions we wait for to ensure test stability